### PR TITLE
Add story1 feature

### PR DESCRIPTION
**Pull request by leaveofabsence**
@jimwheaton can you review this? I'm not sure if I got the inheritance correct

### CODE REVIEW

**Comment by jimwheaton** [comment]
The file is blank. I think you forgot to add the body

File: story1.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Story1
+    attr_accessor :title
+    def initialize(title)
+        @title = title
```
Comment (jimwheaton) [inline story1.rb]: Change this indent to 2 spaces

Comment (leaveofabsence) [inline story1.rb]: Whoops. My editor is configured for 4 spaces

**Comment by jimwheaton** [comment]
Looks good

**Pull request merged**